graph/nullable.md

```diff
@@ -0,0 +1,312 @@
+# Nullable Properties
+
+A nullable property means *only* that the property may have `null` as a value; the "nullability" of a property does not say anything about how a value is set into a property.
+For example, a non-nullable property is *not* required to create a new instance of an entity.
```

Comment:
We need to fix the linting.  Pretty sure there's a rule that warns of a breaking change when adding a non-nullable property.

Reply:
I created [this](https://dev.azure.com/IdentityDivision/Engineering/_workitems/edit/2634761) work item for that. Do you think that work item needs to be created before the guidance is published?

Reply:
Nope.  If anything it's the reverse, because when a rule fails, it should point to guidance.